packages/components/src/components/calendar/views/calendar-days.jsx

```diff
@@ -78,7 +78,7 @@ const getDays = ({
 
         const calendar_events = events.filter(event =>
             // filter by date or day of the week
-            event.dates.find(d => d === date || getDaysOfTheWeek(d) === toMoment(date).day())
```

Comment:
Why are we removing this check?

Reply:
@matin-binary, in another case message not coming by a red dot

Reply:
left as it was